shared/utils/build-all-remediation-roles.py

```diff
@@ -233,6 +248,10 @@ def builder():
                 role_src = generate_role_for_input_content(
                     input_path, benchmark_id, profile_id, args.template
                 )
+
+                if args.extension.endswith("yml") and \
```

Comment:
if it's args.extension shouldn't we test for equality rather than endswith?

Reply:
@mpreisler why wouldn't endswith work?

Reply:
it would but equality is easier to read and makes more sense IMO.

It's sort of like `name="Gahbay"` vs `name.endswith("Gahbay")`. Both work but one makes more sense :-)